packages/whenable/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@agoric/whenable",
+  "version": "0.1.0",
+  "description": "Remote (shortening and disconnection-tolerant) Promise-likes",
+  "type": "module",
+  "main": "src/index.js",
+  "engines": {
+    "node": ">=14.15.0"
+  },
+  "scripts": {
+    "build": "exit 0",
+    "prepack": "tsc --build tsconfig.build.json",
+    "postpack": "git clean -f '*.d.ts*'",
+    "test": "ava",
+    "test:nyc": "exit 0",
+    "test:xs": "exit 0",
+    "lint-fix": "yarn lint:eslint --fix",
+    "lint": "run-s --continue-on-error lint:*",
+    "lint:eslint": "eslint .",
+    "lint:types": "tsc"
+  },
+  "dependencies": {
+    "@agoric/base-zone": "^0.1.0",
+    "@endo/env-options": "^1.1.0",
+    "@endo/eventual-send": "^1.1.0",
+    "@endo/pass-style": "^1.1.0",
+    "@endo/patterns": "^1.1.0",
+    "@endo/promise-kit": "^1.0.2"
+  },
+  "devDependencies": {
+    "@agoric/internal": "^0.3.2",
```

Comment:
This is unused. I just verified locally that everything still works if it is removed. If you don't have immediate plans for it, please remove it, because it will inhibit moving this package to endo, following store and base-zone.

Reply:
FWIW, seeing track-turns and message-breakpoints both duplicated here, I would really like to see a relayering with less duplication. But that would likely only happen after moving to endo.

Reply:
Removed!  And since it broke a dependency cycle I was hampered by, I could now create the Agoric-flavoured whenables in `@agoric/internal/whenable.js`.

Reply:
Awesome! Resolving.